be/src/service/point_query_executor.cpp

```diff
@@ -264,7 +265,7 @@ Status PointQueryExecutor::_lookup_row_key() {
         // Get rowlocation and rowset, ctx._rowset_ptr will acquire wrap this ptr
         auto rowset_ptr = std::make_unique<RowsetSharedPtr>();
         st = (_tablet->lookup_row_key(_row_read_ctxs[i]._primary_key, true, nullptr, &location,
-                                      INT32_MAX /*rethink?*/, rowset_ptr.get()));
+                                      INT32_MAX /*rethink?*/, segment_caches, rowset_ptr.get()));
```

Comment:
point lookup should not use such a segment cache? it's meaningless
you should allow to pass a nullptr here.

Reply:
`_row_read_ctxs` may have multiple keys to lookup，so I think we also need a segment cache here.